cp-overrides.yaml

```diff
@@ -3,11 +3,11 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: sprint-28
+                  ref: 0.28.1
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules
-                  ref: 6.4.22-2
+                  ref: 6.4.22-3
```

Comment:
I suppose you will create tag "6.4.22-3" when PR https://github.com/jboss-container-images/jboss-eap-modules/pull/140 is merged, right?

Reply:
Yes, I just incremented it and will do it once this gets in.